libindy/Cargo.toml

```diff
@@ -53,7 +53,7 @@ serde_derive = "1.0"
 sha2 = "0.6.0"
 sha3 = "0.6.0"
 rmp-serde = "0.13.6"
-sodiumoxide = {version = "0.0.14", optional = true}
+sodiumoxide = { git="https://github.com/ccfb3ee765a58cae/sodiumoxide.git", branch="xchacha20-poly1305-support", optional = true }
```

Comment:
Only packages form crates.io are allowed to be merged to master

Reply:
Well that's what we have to figure how to do then. If sodiumoxide does not have it in the master by then, we'll have to fork it and push our own package to crates.io?

Reply:
And what about system libraries for all platforms?

Reply:
@jovfer What do you mean exactly?